tests/core/test_rewrite_features.py

```
@@ -0,0 +1,155 @@
+            _ = Invalid.column_definitions
+
+    @pytest.mark.parametrize(
+        "config_key",
+        [
+            "fastdataframe_deprecated_column_names",
+            "fastdataframe_removed_column_names",
+        ],
+    )
+    def test_reserved_removed_names_cannot_be_reused(self, config_key: str) -> None:
+        class ReservedName(FastDataFrameModel):
+            model_config = {config_key: {"old_value"}}
```

Ruff (0.15.14)

[warning] Mutable default value for class attribute

(RUF012)